Makefile

```diff
@@ -255,195 +239,3 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
-
-.PHONY: kubectl-slice
```

Comment:
This is not helm specific

Reply:
Gotcha. I'll revert this but wanted to note that this section and the other section that begins with the line `.PHONY: yq` were added as part of the initial Helm functionality commit: https://github.com/ansible/awx-operator/commit/efaa4718eca39043b2465766d425a32da834e160#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

Reply:
Hmmm, I think we should remove both `yq` and `kubectl-slice` since they are not used for anything other than generating Helm charts 🤔

Reply:
@kurokobo is correct lets remove yq and kubectl-slice

Reply:
@TheRealHaoLiu This should be fixed now. Thanks again @kurokobo